spec/helpers.lua

```diff
@@ -226,4 +226,10 @@ local wait = reload_module("spec.internal.wait")
   make_temp_dir = misc.make_temp_dir,
 
   build_go_plugins = cmd.build_go_plugins,
+
+  get_node_id = misc.get_node_id,
```

Comment:
Perhaps we should not export these apis in `helpers`, could we use `misc` directly?

Reply:
I suppose `get_node_id` has more general usages

Reply:
Currently they are only used in rpc mock, we could move it to misc now and expose them in the future.